include/zephyr/drivers/display.h

```diff
@@ -247,6 +247,10 @@ static inline int display_write(const struct device *dev, const uint16_t x,
 	struct display_driver_api *api =
```

Comment:
Please revise you commit title/message, maybe something like:
`include: drivers: display: verify if optional API is implemented`.
IIRC "sanity check" is a bit too vulgar.